.azure-pipelines/linux-conda-CI.yml

```
@@ -14,11 +14,24 @@ jobs:
   strategy:
     matrix:
 
-      Py310-Onnx130-Rt140-Skl121:
+      Py311-Onnx140-Rt140-Skl122:
         do.bench: '0'
         python.version: '3.10'
```

```suggestion
        python.version: '3.11'
```
Map to the matrix name above?